bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/util/ApiFutureUtil.java

```diff
@@ -37,4 +39,26 @@
       Function<F, T> transform) {
     return adapt(Futures.transform(listenableFuture, transform, MoreExecutors.directExecutor()));
   }
+
+  /**
+   * Creates a new {@code ApiFuture} whose value is a list containing the values of all its
+   * successful input futures.
+   *
+   * <p>Similar to {@link Futures#successfulAsList(Iterable)}.</p>
+   *
+   * @param futures a {@link List} containing {@link ApiFuture} tasks.
+   * @return a future that provides a list of the results of the component futures
+   */
+  public static <V> ApiFuture<List<V>> successfulAsList(
```

Comment:
Consider moving this to https://github.com/googleapis/api-common-java/blob/master/src/main/java/com/google/api/core/ApiFutures.java

Reply:
Thanks for the suggestion.
Seems we won't need to use it for now... However, if the need arises I would consider adding that utility in `api-common` directly.